CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Changed
 - Use the TYPO3 Code of Conduct (#430)
+- Move npm tools and config to default locations (#444)
```

Comment:
This needs to move to the top of the list (as the changelog is from newest to oldest).

Reply:
done